chart/values.yaml

```diff
@@ -384,6 +390,12 @@ workers:
       maxSurge: "100%"
       maxUnavailable: "50%"
 
+  # When not set, the values defined on podSecurity.securityContext will be used
```

Comment:
Sorry about that, I did have it in the past and since them after speaking to @jedcunningham I realised that that node was kind of useless. I will update it accordingly.